.github/workflows/altair-app-deploy-staging.yml

```
@@ -16,7 +16,7 @@ jobs:
       whitelisted-accounts: ''
       network: altair
       requires-pop: 'true'
-      subquery-url: ''
+      subquery-url: https://api.subquery.network/sq/centrifuge/pools-altair__Y2Vud
```

Just promoted the staging subquery to production, so it should be:
```suggestion
      subquery-url: https://api.subquery.network/sq/centrifuge/pools-altair
```